SECURITY.md

```diff
@@ -0,0 +1,18 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+To report a vulnerability, submit the issue via Github in the subsystem it applies to.
```

Comment:
Swap the first two lines and combine them as so 
> To report a vulnerability for the Operating System Abstraction Layer (OSAL) please [submit an issue](URL to issue)
>  For general cFS vulnerabilities please [open a cFS framework issue](URL to open an issue at the bundle level)  and see our [top-level security policy](URL to cFS-bundle security.md)

I think we can leave the "other subsystem" part out since they'll see that if they navigate to the bundle repo.